pkg/proc/riscv64_disasm.go

```diff
@@ -0,0 +1,189 @@
+package proc
+
+import (
+	"fmt"
+
+	"github.com/go-delve/delve/pkg/dwarf/op"
+	"github.com/go-delve/delve/pkg/dwarf/regnum"
+	"golang.org/x/arch/riscv64/riscv64asm"
+)
+
+var (
+	errType = fmt.Errorf("unknown type")
```

Comment:
It doesn't look like this is being used anywhere.

Reply:
Deleted.